src/applications/caregivers/components/FormFields/FacilitySearch.jsx

```diff
@@ -305,16 +305,21 @@ const FacilitySearch = props => {
               searchInputError ? 'caregiver-facilities-search-input-error' : ''
             }`}
           >
-            <label
-              htmlFor="facility-search"
+            <p
               className="vads-u-margin-top--0 vads-u-margin-bottom--1"
+              aria-hidden="true"
             >
               {content['form-facilities-search-label']}
-              <span className="vads-u-color--secondary-dark"> (*Required)</span>
-            </label>
+              <span className="vads-u-color--secondary-dark">
+                {' '}
```

Comment:
I'm wondering if this would feel cleaner moving up to the end of L312?

Reply:
Yeah, I agree with you it'd be less awkward.